### DISCUSSION TITLE
pnpmのインストール方法を教えてください

### DISCUSSION THREAD

**Discussion by tkg-reis** [ryoma-yama/lab-gantt#26] [Q&A] [answered]
@ryoma-yama 
表題の通りです

**Comment by ryoma-yama** [comment, marked as answer]
ご質問ありがとうございます。

一番手っ取り早い方法であるNode.jsのcorepack機能を使います。

前提
- Node.js v20以降をインストール済であること

1. 管理者権限でpwshを開いて: `corepack enable pnpm`
2. プロジェクトのpackage.jsonがある場所で: `corepack use pnpm@latest`
   1. **アップデートもこれを使います**

開発者向けのマニュアルにも記載したいと思います。ありがとうございました！

なおcorepack機能は将来的に削除される可能性があります。ご参考までに。

**Reply by tkg-reis** [reply]
@ryoma-yama 
お疲れ様です。
上記でうまくいかないです。特に２の部分。
帰還後に一緒に解決して頂けると嬉しいです

**Reply by tkg-reis** [reply]
@ryoma-yama 
ありがとうございます。解決しました。
よければzennで記事にまとめてみたいと思いますが、下記の部分↓
```
Node.js v20以降をインストール済であること
管理者権限でpwshを開いて: corepack enable pnpm
プロジェクトのpackage.jsonがある場所で: corepack use pnpm@latest
アップデートもこれを使います
```
の部分のエビデンスか情報源のurlとかってわかったりしますか？
引用元で載せたいので！

**Reply by ryoma-yama** [reply]
@tkg-reis 
公式サイトにありますよ。
- [Installation | pnpm](https://pnpm.io/installation#using-corepack)
